e2e/browser/testApp/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["@inrupt/eslint-config-lib", "next/core-web-vitals"]
```

Comment:
This is all we really should have, but this doesn't necessarily work because the testApp depends on prism & lit which are horribly out of date and really shouldn't be being used in e2e tests.